mappings/net/minecraft/entity/SaddleHolder.mapping

```diff
@@ -1,3 +1,3 @@
-CLASS net/minecraft/class_4981
+CLASS net/minecraft/class_4981 net/minecraft/entity/SaddleHolder
```

Comment:
Maybe `on a stick item controllable`?

Reply:
I was thinking more like `ControllableSaddleHolder`

Reply:
need to distinguish from horses mules etc

Reply:
Pigs and striders are not controllable tho, or at least not in the same way as horses for instance

Reply:
this class has some shared logic for boosting with carrot/fungi on a stick and movement etc.
the name had better reflect this fact.